src/solvers/refinement/string_constraint_generator_indexof.cpp

```diff
@@ -373,7 +301,7 @@ exprt string_constraint_generatort::add_axioms_for_last_index_of(
       equal_exprt(str[index], c)));
   axioms.push_back(a3);
 
-  symbol_exprt n=fresh_univ_index("QA_last_index_of", index_type);
+  symbol_exprt n=fresh_univ_index("QA_last_index_of1", index_type);
```

Comment:
Double-check whether the `fresh...` logic will add a numerical suffix that will be confusing when paired with the numerical suffixes specified here, and if so rename